docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: "3.9"
+services:
+  bot:
+    image: krankkkk/dev-tj:latest
+    #image: togetherjava/tjbot:latest
+    volumes:
+      - "./application/config.json:/config.json"
+      - "./application/logs/:/logs"
+    links:
+      - logs
+    environment:
+      - TJ_LOG_TARGET=logs
+  logs:
+    image: krankkkk/dev-logviewer:latest
```

Comment:
Why is your name here?

Reply:
Slipped through, changed back to the `togetherjava`